src/components/cards/MemberCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from "react";
+import Image from "react-bootstrap/Image";
+import "./MemberCard.css";
+
+interface Props {
+  member: BoardMember | TeamMember;
```

Comment:
i think you can redesign this type to make the extra boolean unecessary. check the comments i made in `types.d.ts`

Reply:
synced offline